src/common/NoResults/styled.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+import { ReactComponent as Picture } from "../../images/Picture.svg"
+
+export const StyledHeader = styled.div`
+    margin: 56px;
+`;
+
+export const StyledImages = styled(Picture)`
+    width: 668px;
+    height: 508px;
+    margin-left: 320px;
+`;
```

Comment:
Jeżeli chodzi o obrazek, to na dużym ekranie wygląda dobrze, jest wyśrodkowany i ma dobre wymiary, ale jak zmniejszymy ekran, to obrazek jest ucinany z prawej strony, a w widoku mobilnym nie ma go wcale. Sugerowałbym ustawienie automatycznych marginesów po bokach oraz pokombinowanie z szerokością tego obrazka, żeby się zmniejszał przy zmniejszaniu rozdzielczości ekranu np. stosując width: x% 😇

Reply:
Dokładnie, ale może dlatego, że w Figmie nie ma opcji na telefon.

Reply:
Na telefonie ustawiłbym mniejszą czcionkę (może 14px tak jak "Search results for"), a obrazek lekko powiększył, natomiast na tabletach obrazek nie jest wyśrodkowany. Margin najlepiej ustawić na automatyczny, żeby się sam dopasowywał w zależności od aktualnej rozdzielczości.